pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[build-system]
+requires = ["flit_core >=2,<4"]
+build-backend = "flit_core.buildapi"
+
+[tool.flit.metadata]
+module = "packaging"
+author = "Donald Stufft and individual contributors"
+author-email = "donald@stufft.io"
+home-page = "https://github.com/pypa/packaging"
+description-file = "README.rst"
+requires-python = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*"
```

Comment:
nit: I tend to put classifiers before these. It's a "metadata that is info" then "metadata that has technical ramifications in dependency resolution" kind of a thing.

Reply:
That's not how Furo does it: https://github.com/pradyunsg/furo/blob/main/pyproject.toml 😉 .